src/sosemanuk.rs

```diff
@@ -163,7 +163,7 @@ impl Sosemanuk {
     pub fn new(key: &[u8], nonce: &[u8]) -> Sosemanuk {
         let mut sosemanuk = Sosemanuk { lfsr: [0; 10], fsm_r: [0; 2], subkeys: [0; 100], output: [0; 80], offset: 80 };
  
-        assert!(key.len() >= 16 && key.len() <= 32);
+        //assert!(key.len() >= 16 && key.len() <= 32);
```

Comment:
Is there a reason that this `assert!` is commented out? Should it be removed or un-commented?